src/main/java/Duke.java

```diff
@@ -1,10 +1,18 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static boolean continueRunning = true;
```

Comment:
For a boolean, try to ensure the variable sounds like one. E.g. `isRunning` or `hasRun`. If you are looking to have a setter method, consider something like `void setRunningState(boolean isRunning);`